deployments/properties/aks.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Deploy to a AKS Cluster",
+    "description": "Deploy an application to a Azure Kubernetes Service Cluster",
+    "creator": "Microsoft Azure",
+    "iconName": "azure",
+    "categories": ["Deployment", "AKS", "Kompose", "Helm", "Kustomize", "Kubernetes"]
```

Comment:
Kompose, Helm, kustomize are not currently detected by repo analysis. Here is the list of tech stacks that are detected https://github.com/github-starter-workflows/repo-analysis-partner/blob/main/tech_stacks.yml. You can raise the request to detect these here: https://github.com/github-starter-workflows/repo-analysis-partner/issues by providing how we should detect them.
Also, AKS is not a tech stack, I think we should remove it as it will not be used for suggesting the templates.

Reply:
ok, done
https://github.com/github-starter-workflows/repo-analysis-partner/issues/17
https://github.com/github-starter-workflows/repo-analysis-partner/issues/18
https://github.com/github-starter-workflows/repo-analysis-partner/issues/19